components/block/element.js

```diff
@@ -9,61 +10,119 @@ export default class Block extends LitElement {
   static styles = [style];
 
   static properties = {
-    blockId: { type: Number},
+    loading: {type: Boolean, state: true},
+    blockId: {type: String},
   };
 
-  renderMenu(){
-    return html`
-      <nav>
-        <button @click=${this.edit}>Edit</button>
-        <button @click=${this.moveUp}>Move Up</button>
-        <button @click=${this.moveDown}>Move Down</button>
-        <button @click=${this.delete}>Delete</button>
-      </nav>
-    `;
+  constructor() {
+    super();
+
+    this.loading = false;
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    this.model.on('sidebar_save:success', this.refresh.bind(this));
+  }
+
+  get layout() {
+    return window.parent.Core.g.layout;
+  }
+
+  get model() {
+    return (this.cached_model ||= this.layout.blocks.findWhere({
+      id: this.blockId,
+    }));
+  }
+
+  get parentModel() {
```

Comment:
This entire method is confusing and too complicated so let's simplify it.

getter is named parentModel when we are actually just returning DOM element and right above we have used model to refere to Backbone models.

Use better naming

Reply:
Wholeheartedly agree. Renamed in fe0b4ee3e33c6f4ff7dfae10e69e64f68266fe59